user_guide_src/source/libraries/uri.rst

```diff
@@ -234,6 +234,19 @@ what the values of the segments are. The segments start at 1 being the furthest
 		echo $request->uri->getSegment(2);
 	}
 
+You can also get the different default value for a given segment. Default value is set by using the second parameter of the ``getSegment()`` method.
```

Comment:
"another default" (instead of the different) or "You can also set a different default value for a particular segment by using the second parameter of the ``getSegment()`` method."

Should we add a note about what the actual default value are? Can't seem to find it in this document.

Reply:
I mentioned the default value.